# Request
geometry_msgs/Point position
geometry_msgs/Quaternion orientation  # quaternion (x, y, z, w)
float64 eef_step
float64 speed_factor
---
# Response
bool success
float64 fraction_complete
string message